### ISSUE TITLE
Request: update installer with digital signature to pass Windows Defender validation

### ISSUE THREAD

**Issue by andrew-hill** [ros/solidworks_urdf_exporter#125]
**What versions of the installer have you tried?**
1.6.1

**Have you confirmed that installing by source is successful?**
no

**What version of SolidWorks and SP?**
SolidWorks 2021
Windows 10 21H2

The issue is not with solidworks compatibility, but Windows Defender kicks up a stink because the installer exe is not signed

![image](https://user-images.githubusercontent.com/368167/169431088-7dab832e-c527-4db3-abc3-0acddccd6562.png)

I expect this could be overcome installing from source, but it may be simpler and inspire more confidence from users if the provided installer is digitally signed (e.g. via SignTool.exe in Visual Studio)?

**Comment by brawner** [comment]
Agreed, a digital signature would be quite useful. Unfortunately, I do not currently have a current signing certificate
